SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracerUserInterface.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialEventTracerUserInterface.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
+#include "Interop/Connection/SpatialEventTracer.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "Interop/SpatialClassInfoManager.h"
+#include "SpatialView/EntityComponentId.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialEventTracerUserInterface);
+
+FString USpatialEventTracerUserInterface::CreateSpanId(UObject* WorldContextObject)
```

Comment:
It might be better for these functions to take a TArray of bytes instead because the worker SDK plan to change the structure of spans - would mean we don't need to make a breaking API change here if we could no longer easily represent it as a string

Reply:
Great idea!